src/webui/api/appcontroller.cpp

```diff
@@ -250,6 +250,47 @@ void AppController::preferencesAction()
     data["rss_processing_enabled"] = RSS::Session::instance()->isProcessingEnabled();
     data["rss_auto_downloading_enabled"] = RSS::AutoDownloader::instance()->isProcessingEnabled();
 
+    // libtorrent settings
+    // Async IO threads
+    data["async_io_threads"] = session->asyncIOThreads();
+    // Checking Memory Usage
```

Comment:
Nitpick, only the first word should be capitalized, others in all lowercase. Please recheck PR.

Reply:
IMO it's acceptable that capitalize each word in a phrase, and only capitalize the first word in a sentence

For example 
https://github.com/qbittorrent/qBittorrent/blob/master/src/webui/api/appcontroller.cpp#L167-L208

Reply:
whatever pleases you... but you should know having a uniformed style is better than mixing them and seeing inconsistent styles really hurt my eyes.